package.json

```diff
@@ -44,14 +44,18 @@
     "semver": "1.0.x",
     "pre-commit": "0.0.x"
   },
-  "pre-commit": ["test", "update"],
+  "pre-commit": [
+    "test",
+    "update"
+  ],
   "scripts": {
     "test": "mocha $(find test -name '*.test.js')",
     "qa": "mocha --ui exports $(find test -name '*.qa.js')",
     "update": "node ./bin/update.js",
     "prepublish": "npm run update"
   },
   "dependencies": {
-    "lru-cache": "2.2.x"
+    "lru-cache": "2.2.x",
+    "tmp": "0.0.31"
```

Comment:
Could you add the semver range the dependency so the package receives future patches automatically?